.github/workflows/lighthouse_accessibility.yml

```diff
@@ -27,6 +27,15 @@ jobs:
       - name: Install Third Party Dependencies
         if: startsWith(github.head_ref, 'update-changelog-for-release') == false
         run: python -m scripts.install_third_party_libs
+      - name: Install chrome version 88.0.4324.96-1
```

Comment:
[optiona] should we add this as an "action" in one file and reuse it?

Reply:
done